ocean_lib/ocean/ocean.py

```diff
@@ -212,10 +215,15 @@ def fixed_rate_exchange(self):
             self.web3, get_address_of_type(self.config, "FixedPrice")
         )
 
+    @property
+    @enforce_types
+    def dispenser(self):
+        return Dispenser(self.web3, get_address_of_type(self.config, "Dispenser"))
+
     @enforce_types
     def create_fixed_rate(
         self,
-        erc20_token: ERC20Token,
+        erc20_token: Union[ERC20Token, ERC20Enterprise],
```

Comment:
As ERC20Enterprise is a subclass of ERC20Token, we don't need the double typing.